### PR TITLE
Fixup .gitattributes

### PR DESCRIPTION
**Pull request by obj-obj**
`text=auto` makes git only apply line endings to files it thinks is text, rather than affecting all files, so file types don't have to be specified as binary manually.

https://git-scm.com/docs/gitattributes#Documentation/gitattributes.txt-Settostringvalueauto

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,2 +1 @@
-* text eol=lf
-*.png binary
\ No newline at end of file
+* text=auto eol=lf
```
Comment (kristofbolyai) [inline .gitattributes]: Why put them in the same line though?

**Comment by kristofbolyai** [comment]
Good catch though

File: .gitattributes (hunk 1 above)
Comment (magicus) [inline .gitattributes]: Missing linebreak at EOF

Comment (obj-obj) [reply .gitattributes]: Oops

**Comment by obj-obj** [comment]
You also might want to consider adding a `.editorconfig` (https://editorconfig.org/)

**Review by magicus — APPROVED** [review]

**Comment by magicus** [comment]
> You also might want to consider adding a `.editorconfig` (https://editorconfig.org/)

We have automatic formatting from Spotless. If you care to craft a `.editorconfig` that matches the Spotless formatting so much it seems worth the point, go ahead. Otherwise, it would probably make more sense to add more IntelliJ options, since that is the IDE that all core Artemis developers are using. (Not saying any other IDE is not allowed, just that we have optimized our workflow for IntelliJ).

**Pull request merged**